Sources/Sentry/Public/SentryOptions.h

```diff
@@ -16,6 +16,11 @@ NS_SWIFT_NAME(Options)
 - (_Nullable instancetype)initWithDict:(NSDictionary<NSString *, id> *)options
                       didFailWithError:(NSError *_Nullable *_Nullable)error;
 
+/**
+ * Optional, custom cache directory. Use when default one can not be accessed, e.g. in security environment.
+ */
+@property (nonatomic, strong) NSString *_Nullable customCacheDirectory;
```

Comment:
```suggestion
@property (nonatomic, strong) NSString *_Nullable cacheDirectory;
```

We can set the default in the `SentryOptions.m` to `NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES)`